docs/guide-helm.md

```diff
@@ -0,0 +1,49 @@
+# Using Helm to install NSM on kubernetes cluster
+
+This document will show you how to use `Helm` for `NSM` installation. 
+
+##Helm installation
```

Comment:
You shoudl have space between ## and Helm, like `## Helm installation`. That way it will render better.
Please check the other occurences here too.